main/src/cgeo/geocaching/connector/lc/LCConnector.java

```diff
@@ -108,7 +108,7 @@ public String getExtraDescription() {
 
     @Override
     public boolean supportsSettingFoundState() {
-        return true;
+        return false;
```

Comment:
As false is the default value anyway, you don't need to overwrite the method at all.

See:
https://github.com/cgeo/cgeo/blob/b571526c5ca19046802364637d9715006d2cf68d/main/src/cgeo/geocaching/connector/AbstractConnector.java#L118-L122

Reply:
can I leave it in in case we need to go back and set it to true?

Reply:
I would say yes :-)